packages/documentation-site/patternfly-docs/content/design-guidelines/content/numerics.md

```diff
@@ -26,13 +27,17 @@ PatternFly date and time formats follow the American standard. When localizing,
 
 ## Absolute or relative time
 
-Using **absolute** or **relative** timestamps depends on the context. To represent the exact date and time that an event occurred, use an absolute timestamp following examples provided in the previous [table of date and time formats](#date-and-time-formats).
+Whether you use **absolute** or **relative** timestamps will depend on context. 
+
+To represent the exact date and time that an event occurred, use an absolute timestamp, while referencing the examples provided in the [table of date and time formats](#date-and-time-formats).
 
-To represent how long ago an event occurred, use a relative timestamp. When reporting relative time, follow these examples:
+To represent how long ago an event occurred, use a relative timestamp.
+
+When reporting relative time, align with the following language:
 
 <div class="ws-content-table">
   
-| **Time frame**      | **Usage**              |
+| **Time frame** | **Relative timestamp** |
 |---------------------|------------------------|
 | 0-60 seconds        | Just now               |
 | 1-60 minutes        | 4 minutes ago          |
```

Comment:
I know this isn't guidance you updated just now - but I find the fact that the relative timestamp eventually becomes an absolute time stamp at the 1+ year confusing. Shouldn't it just say 1 year ago, 2 years ago etc? Why would it go to exact time all of a sudden?

Reply:
I also think the guidance on when to use one over the other is confusing still because it's more telling you what relative vs absolute is, instead of saying exactly what context to use one or the other. More specific examples would be helpful to me!